Vizzuality/heco-invest
# Let 468 fe 2 add an explanation and visualization of the priority landscapes to the project page

This PR adds an explanation to the priority landscapes for the impact option  'Priority landscapes' on the project public page

## Testing instructions

When option “HeCo priority landscapes” is selected to see the project’s impact, there will be an explanation of what those are:

“HeCo priority landscapes are geographic spaces of unique biodiversity conditions with sustainability and management plans developed by Herencia Colombia to ensure the provisioning of quality ecosystems”

There will be a way to identify where each mosaic is located geographically

There will be a way to show a map with the mosaics described (see [LET-315: Design - Add a link in the PD formDONE](https://vizzuality.atlassian.net/browse/LET-315) ).
## Tracking

[LET-468](https://vizzuality.atlassian.net/browse/LET-468)
